src/main/java/dukeApp/files/Delete.java

```diff
@@ -0,0 +1,19 @@
+package dukeApp.files;
+import dukeApp.action.Action;
+import java.util.ArrayList;
+
+public class Delete extends Action {
+    public Delete(int num, ArrayList<Task> arrList) {
+        super(num, arrList);
+    }
+
+    /**
+     * Removes a task from the list.
+     *
+     * @return a String statement indicating a task has been removed
+     */
+    public String deleteTask() {
```

Comment:
I think it would be better to separate the ui part (string to display to user) and the logic part (deleting task) to minimise coupling, by returning true/ false instead, then the Ui can use the return value to display the appropriate message.